### PR TITLE
AMLS-5333 | Changes required for AMLS-5333

### PR DESCRIPTION
**Pull request by owen-opencast**
Changes required due to updating AMP models and for loading data from stubs.

## Related / Dependant PRs?

https://github.com/hmrc/amls-frontend/pull/1203
https://github.com/hmrc/amls/pull/270

## How Has This Been Tested?

Spec tests, manual testing

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/controllers/BoughtOrSoldOverThresholdController.scala [hunk 1]
```diff
@@ -65,7 +65,7 @@ class BoughtOrSoldOverThresholdController @Inject()(
         value =>
           for {
             updatedAnswers <- Future.fromTry(request.userAnswers.set(BoughtOrSoldOverThresholdPage, value))
-            _              <- sessionRepository.set(updatedAnswers)
+            _              <- sessionRepository.set(request.internalId, updatedAnswers)
```
Comment (ajchilts) [inline app/controllers/BoughtOrSoldOverThresholdController.scala]: Can we please rename this to `credId' here;

/Users/chilts78/dev/amls-art-market-participant-frontend/app/models/requests/DataRequest.scala

Comment (ajchilts) [reply app/controllers/BoughtOrSoldOverThresholdController.scala]: Both places?

File: app/repositories/SessionRepository.scala [hunk 2]
```diff
@@ -60,21 +58,22 @@ class DefaultSessionRepository @Inject()(
...
-
-    collection.flatMap {
-      _.update(selector, modifier, upsert = true).map {
-        lastError =>
-          lastError.ok
-      }
-    }
-  }
+    return Future(false)
+//    val selector = Json.obj(
```
Comment (ajchilts) [inline app/repositories/SessionRepository.scala]: Can we remove this commented out code?

File: app/controllers/BoughtOrSoldOverThresholdController.scala (hunk 1 above)
Comment (owen-opencast) [reply app/controllers/BoughtOrSoldOverThresholdController.scala]: Done

**Pull request merged**